onfido-java/src/main/java/com/onfido/WorkflowRunManager.java

```diff
@@ -41,4 +42,16 @@ public WorkflowRun create(WorkflowRun.Request request) throws OnfidoException {
   public WorkflowRun find(String workflowRunId) throws OnfidoException {
     return workflowRunParser.parse(get(workflowRunId));
   }
+
+  /**
+   * Retrieves a Workflow Run signed evidence file.
+   *
+   * @param workflowRunId the workflow run id
+   * @return byte array of the evidence file (PDF)
+   * @throws OnfidoException the onfido exception
+   */
+  public byte[] evidence(String workflowRunId) throws OnfidoException {
```

Comment:
Consider using the [FileDownload](https://github.com/onfido/onfido-java/blob/master/onfido-java/src/main/java/com/onfido/api/FileDownload.java) class as in other download endpoints in this library

Reply:
thanks @DavidMealha-Onfido. I added the changes. Could you please have a look?